rpcs3/Emu/RSX/GL/GLTextureCache.h

```diff
@@ -277,6 +291,7 @@ namespace gl
 					init_buffer();
 
 				aa_mode = static_cast<gl::render_target*>(image)->read_aa_mode;
+				ASSERT( managed_texture.get() == nullptr );
```

Comment:
Why space it like that?
```suggestion
				ASSERT(managed_texture.get() == nullptr);
```

Reply:
Agreed, the format of asserts is mixed - some are spaced, some are not, all in the same commit. Should at least be uniform.